backend/diagram/class/IEXCloudPackage.puml

```diff
@@ -0,0 +1,14 @@
+@startuml
+
+package StockMarketData {
+    class StockMarketDataFacade {
```

Comment:
This is the same file as the IEXCloudFacade.puml. Which one or the other be removed?

Reply:
Updated!